recipes/conda-build/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.21.5" %}
+
+package:
+  name: conda-build
+  version: {{ version }}
+
+source:
+  fn: conda-build-{{ version }}.tar.gz
+  url: https://github.com/conda/conda-build/archive/{{ version }}.tar.gz
+  sha256: af90f4d086377bfa9e9d2be590c1d14c79df58937fb12102b67bf0a5e03032c8
+
+build:
+  number: 0
+  script: python setup.py install
+  entry_points:
+    - conda-env = conda_env.cli.main:main
```

Comment:
lol. I copied the `conda-env` recipe over and forgot to change this. In any event, `setuptools` is not being used during the install. So, it shouldn't muck things up I don't think.